programs/create-genesis/genesis_create.cpp

```diff
@@ -769,61 +792,60 @@ struct genesis_create::genesis_create_impl final {
-        auto gls_pk = insert_stat_record(golos_sym, supply, golos_max_supply, issuer_account_name);
+        auto sys_pk = insert_stat_record(sys_supply, system_max_supply, config::system_account_name);
+        auto gls_pk = insert_stat_record(supply, golos_max_supply, issuer_account_name);
 
         // vesting info
         db.start_section(gls_vest_account_name, N(stat), "vesting_stats", 1);
-        table_request tbl{gls_vest_account_name, gls_vest_account_name, N(stat)};
         primary_key_t vests_pk = VESTS >> 8;
-        auto vests_info = mvo
+        db.insert(vests_pk, gls_vest_account_name, mvo
             ("supply", asset(data.total_gests.get_amount(), symbol(VESTS)))
-            ("notify_acc", notify_account_name);
-        db.insert(tbl, vests_pk, vests_info, ram_payer);
+            ("notify_acc", notify_account_name)
+        );
 
         // funds
+        auto balance_obj = [](const asset& x) {
+            return mvo
+                ("balance", x)
+                ("payments", asset(0, x.get_symbol()));
+        };
         const auto n_balances = 3 + 2*data.gbg.size();
         db.start_section(config::token_account_name, N(accounts), "account", n_balances);
         ee_genesis.balances.start_section(config::token_account_name, N(balance), "balance_event", n_balances);
-        auto insert_balance_record = [&](name account, const asset& balance, primary_key_t stat_pk, const ram_payer_info& ram_payer) {
-            db.insert({config::token_account_name, account, N(accounts)}, stat_pk,
-                mvo("balance", balance)("payments", asset(0, balance.get_symbol())), ram_payer);
-            ee_genesis.balances.insert(
-                mvo("account", account)("balance", balance)("payments", asset(0, balance.get_symbol())));
+        auto insert_balance_record = [&](name account, const asset& balance, primary_key_t pk, name ram_payer) {
+            auto record = mvo
+                ("balance", balance)
+                ("payments", asset(0, balance.get_symbol()));
+            db.insert(pk, account, record, ram_payer);
+            ee_genesis.balances.insert(record("account", account));
         };
-        
-        insert_balance_record(config::stake_account_name, _total_staked, sys_pk, ram_payer);
-        insert_balance_record(gls_vest_account_name, gp.total_vesting_fund_steem, gls_pk, ram_payer);
-        insert_balance_record(gls_post_account_name, gp.total_reward_fund_steem, gls_pk, ram_payer);
-        // TODO: internal pool of posting contract
+        insert_balance_record(config::stake_account_name, _total_staked, sys_pk, name());
+        insert_balance_record(gls_vest_account_name, gp.total_vesting_fund_steem, gls_pk, name());
+        insert_balance_record(gls_post_account_name, gp.total_reward_fund_steem, gls_pk, name());
 
         // accounts GOLOS
-        asset total_gls = asset(0, golos_sym);
+        asset total_gls = asset(0, symbol(GLS));
```

Comment:
In Golos dApp `GLS` symbol is missing now.

Reply:
it's GOLOS:
https://github.com/GolosChain/cyberway/blob/3f9c2633b575711bc1f3acf365e6c8a9b47cb325/programs/create-genesis/genesis_create.cpp#L56